step-templates/octopus-register-listening-target.json

```diff
@@ -1,135 +1,135 @@
+  },
+  "Parameters": [
+    {
+      "Id": "f4475bac-5fdb-4d3b-a5de-20dafb8d847e",
+      "Name": "RegisterListeningTarget.Octopus.Base.Url",
+      "Label": "Octopus Base Url",
+      "HelpText": "The base url of your Octopus Deploy instance.  Example: https://samples.octopus.app",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "d25ad4b7-9281-4ee0-833d-b07f0e4b5b19",
+      "Name": "RegisterListeningTarget.Octopus.Api.Key",
```

Comment:
This looks like a new ID, did you paste in the entire JSON body or just the script?